js/config/scoring.js

```diff
@@ -53,30 +63,38 @@ YoastSEO.AnalyzerScoring = function( i18n ) {
                 {
                     min: 3.5,
                     score: -50,
-                    text: i18n.dgettext('js-text-analysis', "The keyword density is %1$f%, which is way over the advised 2.5% maximum, the focus keyword was found %1$d times.")
+
+                    /* translators: %1$f expands to the keyword density percentage, %2$d is the amount of times the keyword is present */
```

Comment:
"%2$d is the amount of times the keyword is present" --> "%2$d expands to the number of times the keyword is found"